arkouda/pdarrayclass.py

```diff
@@ -4041,6 +4064,11 @@ def fmod(dividend: Union[pdarray, numeric_scalars], divisor: Union[pdarray, nume
     -------
     pdarray
         Returns an array that contains the element-wise remainder of division.
+
+    Raises
+    ------
+    TypeError
+        Raised if either dividend or divisor fails typechecking
```

Comment:
Could this be more meaningful?  Perhaps: "Raised if either dividend or divisor is not a scalar or pdarray"?

Reply:
Done.  It is now more specific.

Reply:
Thanks!